packages/ember-metal/lib/property_events.js

```diff
@@ -108,6 +111,13 @@ function propertyDidChange(obj, keyName) {
   }
 
   markObjectAsDirty(m);
+
+  if (isEnabled('ember-glimmer-detect-two-way-reflush') ||
+      isEnabled('ember-glimmer-allow-two-way-reflush')) {
+    if (has('glimmer-runtime')) {
```

Comment:
seems bad to do this as a runtime check since this is very hot path; should probably just make sure we only enable the flags with glimmer, (or if that's not possible, at least make the functions no-op without glimmer and call it here unconditionally)